pythonforandroid/toolchain.py

```diff
@@ -1093,6 +1094,8 @@ def _build_package(self, args, package_type):
                     gradle_task = "assembleRelease"
                 elif package_type == "aab":
                     gradle_task = "bundleRelease"
+                else:
+                    gradle_task = "assembleRelease"
```

Comment:
Looks the same of `package_type == "apk"`, how about avoiding the `else` here ?

Reply:
updated the file to match the request